frontend/packages/gitops-plugin/src/components/details/deploymentHistory/GitOpsDeploymentHistory.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Stack } from '@patternfly/react-core';
+import { GitOpsEnvironment } from '../../utils/gitops-types';
+
+interface GitOpsDeploymentHistoryProps {
+  customData: {
+    envs: GitOpsEnvironment[];
+    appName: string;
+  };
+}
+
+const GitOpsDeploymentHistory: React.FC<GitOpsDeploymentHistoryProps> = ({
+  customData: { envs, appName },
+}) => {
+  return (
+    <div className="odc-gitops-details">
+      {_.map(
+        envs,
+        (env) =>
+          env && (
+            <Stack>
```

Comment:
Looks WIP-esk... but don't forget to put React `key`s.

Reply:
this is the area that Keith will be contributing. Likely most of this will be overwritten.

Reply:
Is this something that was intended to be removed?